apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -4953,6 +4953,9 @@
     "FeeTooHigh": {
       "title": "Network fees are above 10% of the amount"
     },
+    "DustLimit": {
+      "title": "Amount is lower than dust limit"
+    },
```

Comment:
Neat :ok_hand:  you just need to do the same on LLM, [here](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-mobile/src/locales/en/common.json).